bpaotu/bpaotu/views.py

```diff
@@ -495,6 +495,226 @@ def str_none_blank(v):
         return csv_fd.getvalue()
 
 
+
+
+@require_http_methods(["GET"])
+def otu_biom_export(request):
+    SAMPLE_RUN_VAL = 1
+    TRIAL_SWITCH = False
+
+
+
+    def val_or_empty(obj):
+        if obj is None:
+            return ''
+        return obj.value
```

Comment:
I think we have this function elsewhere; just move it into `util.py` (or wherever) and import it

Reply:
Fixed.